### PR TITLE
Adds button and header styling

### PR DESCRIPTION
**Pull request by jprittie**
Hello... this feels quite hacky compared to what Jocelyn did and I won't be offended if you want a major overhaul. I'm thinking I probably should have used flexbox in the header?

We can also probably find a better background image. Normally I have good luck with unsplash, but couldn't find anything that said "liquor store." I also added a red svg circle we can use with the shopping cart. It's just hanging out in the img folder... I haven't imported or styled it yet.

Jen!! I don't know how to add a comment directly under yours, so I'm editing. This isn't hacky AT ALL. It's good. And you can definitely have flexbox in the header...or not. It's whatever style you like and what works for you. Good work. -JJ

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -6,3 +6,46 @@
...
+  min-width: 100px;
+  height: 40px;
+  font-size: 1.1em;
+  color: #FFF;
+  background-color: #4197e2;
+  border-radius: 10px;
+  border-width: none;
+  border-style: none;
+  border-image: none;
+  border-color: none;
```
Comment (astr0-4) [inline src/App.css]: I think border-style, image, and color all have default values of none so you could remove these

File: src/components/Header/Header.js [hunk 2]
```diff
@@ -1,13 +1,21 @@
 import React from 'react'
 import './Header.css'
 import shopping_cart from '../../img/shopping_cart.svg'
+import cheers from '../../img/cheers.jpg'
 import SearchBar from '../SearchBar'
 
+const headerImageStyle = {
+	width: "100%",
+  backgroundImage: 'url(' + cheers  + ')',
```
Comment (astr0-4) [inline src/components/Header/Header.js]: you could use the template literals style to write this: \` url(${cheers}) \`

File: src/App.css [hunk 3]
```diff
@@ -6,3 +6,46 @@
...
+  max-width: 80px;
+  min-width: 100px;
+  height: 40px;
+  font-size: 1.1em;
+  color: #FFF;
+  background-color: #4197e2;
+  border-radius: 10px;
+  border-width: none;
+  border-style: none;
+  border-image: none;
```
Comment (jocelynjeffrey) [inline src/App.css]: you can take this line out - it's not doing anything here

File: src/App.css [hunk 4]
```diff
@@ -6,3 +6,46 @@
 ul {
   list-style: none;
 }
+
+button {
+  width: 12%;
+  max-width: 80px;
+  min-width: 100px;
+  height: 40px;
```
Comment (jocelynjeffrey) [inline src/App.css]: your max-width is less than your min-width here; your min will cancel out your max. You can probably ditch the max-width

File: src/App.css [hunk 5]
```diff
@@ -6,3 +6,46 @@
...
+  -ms-transform: scale(1.03, 1.03); /* IE 9 */
+   -webkit-transform: scale(1.03, 1.03); /* Safari */
+   transform: scale(1.03, 1.03);
+}
+
+*::-webkit-input-placeholder {
+    color: #000;
+    font-size: 1.1em;
+}
+*:-moz-placeholder {
```
Comment (jocelynjeffrey) [inline src/App.css]: I think this syntax is deprecated. You should be okay using only ::moz-placeholder from above

File: src/components/Header/Header.css [hunk 6]
```diff
@@ -1,24 +1,25 @@
 .header {
   background-color: #000;
   height: 150px;
-  padding: 100px;
+  padding-top: 100px;
```
Comment (jocelynjeffrey) [inline src/components/Header/Header.css]: you can combine these with shorthand! `padding: 100px 0;` The shorthand takes up to 4 options, one for each side (set clockwise, from the top). If you only give 2, it uses the 1st as top and bottom and the 2nd as left and right.

File: src/components/SearchBar/SearchBar.css [hunk 7]
```diff
@@ -0,0 +1,24 @@
+
+.searchbar .text-input {
+  width: 40%;
+  max-width: 300px;
+  min-width: 80px;
+  height: 40px;
+  margin-right: 20px;
+  padding: 0 0 0 20px;
```
Comment (jocelynjeffrey) [inline src/components/SearchBar/SearchBar.css]: nice!

File: src/components/SearchBar/SearchBar.css [hunk 8]
```diff
@@ -0,0 +1,24 @@
+
+.searchbar .text-input {
+  width: 40%;
+  max-width: 300px;
+  min-width: 80px;
```
Comment (jocelynjeffrey) [inline src/components/SearchBar/SearchBar.css]: I don't think you ever hit this size, you probably can remove this.

File: src/components/SearchBar/SearchBar.css [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+  height: 40px;
+  margin-right: 20px;
+  padding: 0 0 0 20px;
+  border-radius: 10px;
+  color: #000;
+  font-size: 1em;
+  background-color:rgba(255,255,255, 0.5);
+  border-width: none;
+  border-style: none;
+  border-image: none;
```
Comment (jocelynjeffrey) [inline src/components/SearchBar/SearchBar.css]: can remove

File: src/components/SearchBar/SearchBar.css [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+  border-color: none;
+}
+
+.searchbar .text-input:focus, .searchbar .text-input:active {
+  border-width: none;
+  border-style: none;
+  border-image: none;
```
Comment (jocelynjeffrey) [inline src/components/SearchBar/SearchBar.css]: can remove border-image and border-style

**Review by jocelynjeffrey — COMMENTED** [review]
I left a few comments for updates, but great work. It looks so good!

File: src/components/SearchBar/SearchBar.css [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+  border-color: none;
+}
```
Comment (astr0-4) [inline src/components/SearchBar/SearchBar.css]: I think here as well you can remove border-style, image, and color

File: src/App.css [hunk 12]
```diff
@@ -6,3 +6,46 @@
 ul {
   list-style: none;
 }
+
+button {
+  width: 12%;
+  max-width: 80px;
+  min-width: 100px;
```
Comment (astr0-4) [inline src/App.css]: here your `max-width` is smaller than your `min-width`

**Review by astr0-4 — CHANGES_REQUESTED** [review]
this looks amazing @jprittie!!! 
just some small fixes and it's good to go

File: src/App.css [hunk 13]
```diff
@@ -6,3 +6,46 @@
 ul {
   list-style: none;
 }
+
+button {
```
Comment (jprittie) [inline src/App.css:10]: Comment

**Review by jocelynjeffrey — APPROVED** [review]

**Review by astr0-4 — APPROVED** [review]

**Pull request merged**